### DISCUSSION TITLE
I would like to ask you to correct the garbled characters

### DISCUSSION THREAD

**Discussion by y-Bobby** [hbb20/CountryCodePickerProject#572] [General]
I would like to ask you to correct the garbled characters．
Japanese Res.
ccp_japanese.xml

NG
-------
        <country
            name="七面鳥"
            english_name="Turkey"
            name_code="tr"
            phone_code="90" />
----
OK
----
        <country
            name="トルコ"
            english_name="Turkey"
            name_code="tr"
            phone_code="90" />
----

NG
----
        <country
            name="出会い"
            english_name="Réunion"
            name_code="re"
            phone_code="262" />
----

OK
----
        <country
            name="マイヨット島"
            english_name="Réunion"
            name_code="re"
            phone_code="262" />
----

**Comment by hbb20** [comment]
Sorry. Just got to see this. Will release a new version soon with the fix.

**Comment by Ashish7772** [comment]
we want to change only flag view in circle but its not working why?

**Reply by hbb20** [reply]
You can do the custom layout and force the circular flag. This should work technically but there are some considerations you will have to do.
1. The flags in the library are rectangles so simply clipping in a circle won't give the best experience 
2. Without Circular flag style some flags might loss critical elements during clipping.
